.doc_gen/templates/zonbook/utility/sdk_api_examples.xml

```diff
@@ -5,69 +5,37 @@
 {{- if isSnapshot}}
     {{- $include_docs = ""}}
 {{- end}}
-{{- if $examples.ApiExamples}}
-<section id="{{$sdk_prefix}}_code_examples_categorized" role="topic">
+<para role="topiclist-abbrev">Services</para>
+{{- range $service, $svc_examples := $examples.ApiExamples}}
+{{- if $svc_examples.ServiceEntity.Short}}
+{{- $basics := index $svc_examples.CategorizedExampleSets "Basics"}}
+{{- $actions := index $svc_examples.CategorizedExampleSets "Actions"}}
+{{- $scenarios := index $svc_examples.CategorizedExampleSets "Scenarios"}}
+<section id="{{$sdk_prefix}}_{{$svc_examples.ServiceSlug}}_code_examples" role="topic">
     <info>
-        <title id="{{$sdk_prefix}}_code_examples_categorized.title">Actions and scenarios using {{$examples.SdkEntity.Short}}</title>
-        <titleabbrev id="{{$sdk_prefix}}_code_examples_categorized.titleabbrev">Actions and scenarios</titleabbrev>
+        <title id="{{$sdk_prefix}}_{{$svc_examples.ServiceSlug}}_code_examples.title">{{$svc_examples.ServiceEntity.Short}} examples using {{$examples.SdkEntity.Short}}</title>
+        <titleabbrev id="{{$sdk_prefix}}_{{$svc_examples.ServiceSlug}}_code_examples.titleabbrev">{{$svc_examples.ServiceEntity.Short}}</titleabbrev>
         <abstract>
-            <para>The following code examples show how to implement common scenarios and perform individual service actions by using
-                {{$examples.SdkEntity.Short}} with &AWS-services;.</para>
+            <para>Code examples that show how to use {{$examples.SdkEntity.Long}} with {{$svc_examples.ServiceEntity.Short}}.</para>
         </abstract>
     </info>
-    <para>The following code examples show how to perform actions and implement common scenarios by using
-        the {{$examples.SdkEntity.Long}} with &AWS-services;.</para>
-    {{- template "note_example_types" makeSlice 1 1 0}}
-    <para role="topiclist-abbrev">Services</para>
+    <para>The following code examples show you how to perform actions and implement common scenarios by using
+        the {{$examples.SdkEntity.Long}} with {{$svc_examples.ServiceEntity.Short}}.</para>
+    {{- template "note_example_types" makeSlice 0 $basics.Examples $actions.Examples $scenarios.Examples}}
+    <para>Each example includes a link to GitHub, where you can find
```

Comment:
Same comment as above, could we say a link to "complete source code", maybe that will help customers to want to go there.

Reply:
Done